From e3086f82502a201ba263334bc951d38cb46024c7 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Sat, 5 Jun 2021 09:52:53 -0700
Subject: [PATCH 116/196] Update testnet block nonce and time for launching
 tomorrow (#189)

Set testnet genesis block timestamp to June 5th 12PM PST.
---
 src/chainparams.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/chainparams.cpp b/src/chainparams.cpp
index 03a9aef8c..ae745ab4b 100644
--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -230,12 +230,12 @@ public:
         m_assumed_chain_state_size =
             ChainParamsConstants::TESTNET_ASSUMED_CHAINSTATE_SIZE;
 
-        genesis = CreateGenesisBlock(0x1c100000, 1619854081, 159684556417ull);
+        genesis = CreateGenesisBlock(0x1c100000, 1622919600, 532395334422ull);
         consensus.hashGenesisBlock = genesis.GetHash();
         assert(genesis.GetSize() == 379);
         assert(consensus.hashGenesisBlock ==
-               uint256S("0000000003b908ce4df90b91912fd02f32e971a7782afd5f6892d5"
-                        "99169432ac"));
+               uint256S("00000000080a6c9633aae9d24b9acda10d7e6b028e7aa714069798"
+                        "d18ca7bad1"));
         assert(genesis.hashMerkleRoot ==
                uint256S("37f392d88f70cdada6d366a25a7ef90b6711bf2d6b5ffea4f39727"
                         "dcb90af34c"));
-- 
2.39.5 (Apple Git-154)

